core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -232,6 +263,29 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     }
   }
 
+  calculatePanelHeight(): void {
+    setTimeout(()=>{
+      this.isContentOverflowing = (
+        this.contentPanel.elementRef.nativeElement.offsetHeight >
+        this.contentContainer.nativeElement.offsetHeight);
+      this.isTranslationOverflowing = (
+        this.translationPanel.elementRef.nativeElement.offsetHeight >
+        this.translationContainer.nativeElement.offsetHeight);
+    }, 0);
+  }
+
+  ngAfterViewInit(): void {
+    this.calculatePanelHeight();
+  }
+
+  toggleExpansionState(tab: string): void {
+    if (tab === 'content') {
```

Comment:
Can the expansion state be made a TypeScript enum?

Reply:
Done